src/test/mod.rs

```diff
@@ -12,8 +12,11 @@ use quote::quote;
 use syn::Result;
 
 fn test_derive(input: TokenStream, expected: TokenStream) -> Result<()> {
-	let left = crate::derive_where_internal(input)?.to_string();
-	let right = quote! { #expected }.to_string();
+	let left = crate::derive_where_internal(input.clone())?.to_string();
+	// FIXME this currently means, that `input_without_derive_where_attributes` is
+	// not actually tested.
+	let item = crate::input_without_derive_where_attributes(syn::parse2(input)?);
+	let right = quote! {#item #expected}.to_string();
```

Comment:
We would need to change every unit test to include the item without derive_where attributes to remove this.

Reply:
I'm not following, can you explain?

Reply:
well the thing I documented in the FIXME, if we want to not use `input_without_derive_attributes`.